ast_comments.py

```diff
@@ -72,7 +77,7 @@ def _enrich(source: Union[str, bytes], tree: ast.AST) -> None:
 
         attr = getattr(target_node, target_attr)
         attr.append(c_node)
-        attr.sort(key=lambda x: (x.end_lineno, not isinstance(x, Comment)))
+        attr.sort(key=lambda x: (x.end_lineno, isinstance(x, Comment)))
```

Comment:
breaking change: Comments are no longer sorted before other nodes in the same line